dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.h

```diff
@@ -183,6 +184,8 @@ class SegmentReadTaskPool : private boost::noncopyable
         , unordered_input_stream_ref_count(0)
         , exception_happened(false)
         , mem_tracker(current_memory_tracker == nullptr ? nullptr : current_memory_tracker->shared_from_this())
+        , block_slot_limit(std::max(num_streams_, 3))
+        , active_segment_limit(std::max(num_streams_, 2))
```

Comment:
Thread number of a SegmentReadPool object is the same as the number of CPU logical core of a CPU NUMA node.
Why `active_segment_limit` and `block_slot_limit` is set to `num_streams_` which mainly equal to half of the number of CPU logical core? When there are two NUMA node, they are the same, but what if there are more than two NUMA node?

Reply:
get